commons/passive/phone/phone_bluetooth_devices_scanned.avsc

```diff
@@ -0,0 +1,11 @@
+{
+  "namespace": "org.radarcns.passive.phone",
+  "type": "record",
+  "name": "PhoneBluetoothDevicesScanned",
+  "doc": "Phone Bluetooth devices info.",
+  "fields": [
+    {"name": "time", "type": "double", "doc": "Device timestamp in UTC (s)."},
+    {"name": "timeReceived", "type": "double", "doc": "Device receiver timestamp in UTC (s)."},
+    {"name": "macAddressHash", "type": ["null", "bytearray"], "default": null, "doc":"Hash of Nearby Bluetooth device MAC address"}
```

Comment:
If we are taking a snapshot of the Bluetooth environment then presumable we should have a set of hashes (so probably pluralise the doc and name?
```suggestion
    {"name": "macAddressHashes", "type": ["null", "bytearray"], "default": null, "doc":"Hash of Nearby Bluetooth device MAC addresses"}
```

Reply:
thinking again perhaps it is better not to collect the set of devices in a single payload for the snapshot?

Reply:
I believe the plan is to send this record for each bluetooth device. That's why it was separated from the bluetooth schema.

Reply:
Yep, I saw Joris's comment after I put that in. I agree it's a better proposition.

Reply:
So, do I make these changes or not?

Reply:
No, should not be needed.